libs/components/src/lib/alert/alert.ts

```diff
@@ -91,29 +95,41 @@ export class Alert extends FoundationElement {
 	 */
 	@attr connotation?: AlertConnotation;
 
+	/**
+	 *
+	 * @internal
+	 */
+	controlEl?: HTMLDivElement;
+
 	/**
 	 * indicates whether the alert is open
 	 *
 	 * @public
 	 * HTML Attribute: open
 	 */
-	@attr({ mode: 'boolean'	}) open = false;
+	@attr({ mode: 'boolean' }) open = false;
 	openChanged(oldValue: boolean, newValue: boolean): void {
 		if (oldValue === undefined) return;
 		this.$emit(newValue ? 'open' : 'close');
 		this.#setupTimeout();
+		if (newValue) {
+			this.style.display = 'block';
```

Comment:
```suggestion
			this.style.display = 'inline';
```
the initial display of a host is inline.
I don't see a justification here to change it to block (even if it does not effect on alert visually)